test/fixtures/lambda_graphql.js

```diff
@@ -1,35 +1,25 @@
-const convert = require('koa-convert');
 const Koa = require('koa');
-const koaBodyParser = convert(require('koa-better-body')({ fields: 'body' }));
-const Router = require('koa-router');
 const serverless = require('serverless-http');
 
-const { graphqlKoa } = require('apollo-server-koa');
-const { makeExecutableSchema } = require('graphql-tools');
+const { ApolloServer, gql } = require('apollo-server-koa');
 
 const app = new Koa();
-const router = new Router();
 
-const schema = makeExecutableSchema({
+const graphql = new ApolloServer({
+  context: ({ctx}) => ({
+    header: ctx.request.get('test-header')
+  }),
   resolvers: {
     Query: {
       value: (obj, args, context, info) => args.prompt + ': ' + context.header
     }
   },
-  typeDefs: `
+  typeDefs: gql`
     type Query {
       value (prompt: String!): String!
     }
   `
 });
 
-const graphql = graphqlKoa((context) => ({
-  context: {
-    header: context.request.get('test-header')
-  },
-  schema
-}));
-
-router.post('/', koaBodyParser, graphql);
-app.use(router.routes());
+graphql.applyMiddleware({ app, path: '/' });
```

Comment:
It feels like this change will require an adoption by the users of lambda-tools. Do you agree that this change justifies a major version bump? If so, would it be worth updating the package.json version to the next major version and use the `.beta` qualifier before it's released?

Reply:
This just changes our test coverage and doesn't really change the API/features for consumers. In general, `lambda-tools` should theoretically be tool agnostic so I don't think any version bump is required for this change. Did I miss something?

Reply:
It was my mistake. I was imaging that lambda-tools was exposing the `apollo-server-koa` to consumers. Because of the way the github diff compressed the differences, and because I wasn't diligent, I didn't notice it was dev only.